libraries/AP_InertialNav/AP_InertialNav_NavEKF.cpp

```diff
@@ -14,34 +14,23 @@
 */
 void AP_InertialNav_NavEKF::update(float dt)
 {
-    // get the NE position relative to the local earth frame origin
-    Vector2f posNE;
-    if (_ahrs_ekf.get_relative_position_NE_origin(posNE)) {
-        _relpos_cm.x = posNE.x * 100; // convert from m to cm
-        _relpos_cm.y = posNE.y * 100; // convert from m to cm
-    }
-
-    // get the D position relative to the local earth frame origin
-    float posD;
-    if (_ahrs_ekf.get_relative_position_D_origin(posD)) {
-        _relpos_cm.z = - posD * 100; // convert from m in NED to cm in NEU
-    }
-
-    // get the velocity relative to the local earth frame
-    Vector3f velNED;
-    if (_ahrs_ekf.get_velocity_NED(velNED)) {
-        _velocity_cm = velNED * 100; // convert to cm/s
-        _velocity_cm.z = -_velocity_cm.z; // convert from NED to NEU
-    }
 }
 
 /**
  * get_position - returns the current position relative to the home location in cm.
  *
  * @return
  */
-const Vector3f &AP_InertialNav_NavEKF::get_position(void) const 
+const Vector3f &AP_InertialNav_NavEKF::get_position(void) const
 {
+    // get the NED position relative to the local earth frame origin
```

Comment:
I think we want to get rid of all these calls to AP_InertialNav so let's not spend time changing the internals of AP_InertialNav.